File: assignment-10/qustion4.c
#include <stdio.h>

int RangeSumEven(int iStart,int iEnd){

    if(iStart>iEnd){
        printf("Invalid range");
    }

    int iCnt=0;
    int iSum=0;

    for(iCnt=iStart;iCnt<=iEnd;iCnt++){
        if((iCnt % 2)==0){
            printf("%d",iCnt);
        iSum+=iCnt;
        }
    }
    return iSum;
}
int main()
{
    int iValue1=0,iValue2=0,iRet=0;

    printf("Enter starting point");
    scanf("%d",&iValue1);

    printf("Enter ending point");
    scanf("%d",&iValue2);

    if(iValue1>iValue2 || iValue1<0 ||iValue2<0){
        printf("Invalid range");
        return 1;
    }

    iRet=RangeSumEven(iValue1,iValue2);

    printf("Addition is %d",iRet);

    return 0;
}

File: assignment-35/qustion4.java
import java.util.*;

class Number{
    public int Arr[];
    public int iRStart;
    public int iREnd;

    public Number(int x[],int a,int b){
        this.Arr=x;
        this.iRStart=a;
        this.iREnd=b;
    }

      public void Display()
      {
        int iCnt=0;
        for(iCnt=0;iCnt<Arr.length;iCnt++){
            if(Arr[iCnt]>=iRStart && Arr[iCnt]<=iREnd){
                System.out.print(Arr[iCnt]+"\t");
            }
        }
      }
}
class qustion4{
    public static void main(String args[]){
        Scanner obj=new Scanner(System.in);
        int iSize=0;
        int iCnt=0;
        int iStart=0;
        int iEnd=0;
        
        System.out.println("Enter the number:");
        iSize=obj.nextInt();

        System.out.println("Enter starting Rang no:");
        iStart=obj.nextInt();

        System.out.println("Enter Ending Range no:");
        iEnd=obj.nextInt();

        int Arr[]=new int[iSize];
        System.out.println("Enter the element:");
        for(iCnt=0;iCnt<iSize;iCnt++){
            Arr[iCnt]=obj.nextInt();
        }

        Number nobj=new Number(Arr, iStart,iEnd);
        nobj.Display();

        obj.close();
    }
}